class.json-api-endpoints.php

```diff
@@ -1842,7 +1842,7 @@ function handle_media_sideload( $url, $parent_post_id = 0, $type = 'any' ) {
 			return false;
 
 		// if we didn't get a URL, let's bail
-		$parsed = @parse_url( $url );
+		$parsed = @wp_parse_url( $url );
```

Comment:
Let's remove output silencing here because it's already done inside `wp_parse_url`.